lldb/include/lldb/Symbol/SaveCoreOptions.h

```diff
@@ -56,7 +57,7 @@ class SaveCoreOptions {
   std::optional<lldb_private::FileSpec> m_file;
   std::optional<lldb::SaveCoreStyle> m_style;
   lldb::ProcessSP m_process_sp;
-  std::unordered_set<lldb::tid_t> m_threads_to_save;
+  std::unordered_map<lldb::tid_t, lldb::ThreadSP> m_threads_to_save;
```

Comment:
You can remove this change right? You can fetch the threads from the `m_process_sp` by `lldb::tid_t`

Reply:
Yeah we can, good call out.